src/v0/sources/shopify/transform.js

```diff
@@ -208,11 +213,20 @@ const isIdentifierEvent = (event) =>
   ['rudderIdentifier', 'rudderSessionIdentifier'].includes(event?.event);
 const processIdentifierEvent = async (event, metricMetadata) => {
   if (useRedisDatabase) {
+    const cartToken =
+      typeof event.cartToken === 'string' ? event.cartToken.split('?')[0] : event.cartToken;
+    logger.info(`{{SHOPIFY::}} writeKey: ${metricMetadata.writeKey}, cartToken: ${cartToken}`, {
```

Comment:
@anantjain45823 writekey shouldn't be logged, instead use other identifier like sourceId.

Reply:
we don't have any other identifier other than write key, @utsabc thoughts?